modules/Core/collections/panel/RecentPunishments.php

```diff
@@ -99,7 +99,7 @@ public function getContent(): string {
                         'staff_avatar' => $staff_user->getAvatar(),
                         'staff_uuid' => Output::getClean($staff_user->data()->uuid),
                         'staff_profile' => URL::build('/panel/user/' . Output::getClean($staff_user->data()->id) . '-' . Output::getClean($staff_user->data()->username)),
-                        'time' => ($item->created ? $timeago->inWords(date('Y-m-d H:i:s', $item->created), $this->_language->getTimeLanguage()) : $timeago->inWords($item->infraction_date, $this->_language->getTimeLanguage())),
+                        'time' => ($item->created ? $timeago->inWords(date(DATE_FORMAT, $item->created), $this->_language->getTimeLanguage()) : $timeago->inWords($item->infraction_date, $this->_language->getTimeLanguage())),
                         'time_full' => ($item->created ? date('d M Y, H:i', $item->created) : date('d M Y, H:i', strtotime($item->infraction_date))),
```

Comment:
I think this is the format we need to replace (`d M Y, H:i`), and not `Y-m-d H:i:s`